README.md

```diff
@@ -94,59 +96,59 @@ A simple script "run.py" is provided for you to run a single test or a regressio
 You can use --help to get the complete command reference:
 
 ```bash
-python3 run.py --help
+run --help
```

Comment:
please add an example how to run with the existing approach. just want to make sure user understand there are two ways to run it.